src/App.js

```diff
@@ -92,7 +85,7 @@ class App extends React.Component {
 
       this.setState({
         result: finalResult + "",
-        lastFormula: nextEquation,
+        lastEquation: nextEquation,
```

Comment:
@grimxreaper the commit message doesn't match the code changes. the commit message says "repeat operation every time equals is pressed", That was already implemented, the code change fixes some bugs and enables some tests. perhaps a better commit message would be something like "enable equals pressed repeatedly tests, fix bugs"

Reply:
It wasn't technically implemented because it didn't work, those tests weren't passing until these changes were made (they were being skipped in the previous commits while I was working on fixing the bugs to make them pass) which is why they passed this time. Is there a typo in this?  "enable equals pressed repeatedly tests, fix bugs" or am I not understanding what you mean by _repeatedly tests_...

Reply:
I would expect to find the implementation for "repeat operation every time equals is pressed" in this commit based on the wording, not a bug fix for it and enabling its tests. this is a tricky commit message to write and you're right that my suggestion was unclear, "enable tests for equals pressed repeatedly, fix bugs" is clearer

Reply:
it was definitely implemented, there was code that expected to do what the commit message suggested, it just wasn't working.

Reply:
Ohhh okay, well I see, but that technically is the working implementation for it but I see what you mean, totally agree. 😂 It's tricky because I had to introduce the function for it (handleDoubleEquals) before being able to making the commit for it working properly.